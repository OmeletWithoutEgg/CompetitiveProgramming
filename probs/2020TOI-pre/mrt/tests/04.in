43
17 3 11 7 16 3 9
17 4 1 4 8 15 6
43 29 22 29 22 9 25
31 22 4 16 22 4 30
31 9 20 20 4 26 6
59 7 29 57 40 23 12
59 16 55 11 21 36 38
67 51 66 60 6 18 60
19 17 6 0 14 13 7
13 9 1 7 11 10 5
71 21 60 5 52 37 21
2 0 1 1 0 0 1
43 1 3 42 42 28 28
41 35 13 26 34 9 14
43 8 1 11 42 40 6
71 67 23 39 55 7 14
11 5 9 0 8 10 5
67 29 33 33 55 36 37
53 0 45 35 34 10 41
23 13 5 3 18 4 12
29 2 24 26 7 0 13
59 16 40 19 51 9 38
71 53 66 58 34 29 52
67 50 46 35 60 39 12
19 6 18 3 14 1 14
11 3 8 9 4 6 0
73 26 58 34 60 71 12
5 1 1 0 1 4 2
41 11 38 1 3 21 31
23 15 1 2 19 16 17
11 10 4 4 8 0 2
7 2 6 3 2 5 6
37 5 8 22 31 24 22
61 55 1 11 31 45 24
17 10 3 7 12 10 7
67 49 57 20 27 42 52
71 51 54 38 8 40 63
53 7 40 19 6 32 38
3 2 1 0 1 2 2
37 25 31 21 30 8 22
41 17 14 1 27 17 38
53 52 23 42 44 17 34
3 2 1 0 1 1 2
