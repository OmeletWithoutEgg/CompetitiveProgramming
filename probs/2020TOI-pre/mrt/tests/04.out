9 11
0 1
9 25
8 17
23 5
25 31
27 13
18 60
10 13
0 12
35 54
0 1
17 17
1 7
3 3
55 40
10 5
34 49
18 43
6 0
7 27
38 58
32 33
3 42
15 14
10 9
59 69
2 4
39 7
13 1
1 9
3 2
2 10
9 35
16 14
2 37
57 69
9 14
2 2
4 11
31 22
9 52
0 1
